From 7b1b73f1b05a87dcc2b9b1abf2dff0dd966f90d8 Mon Sep 17 00:00:00 2001
From: Sai Vemuri <svemuri@redhat.com>
Date: Wed, 12 Nov 2014 19:08:14 -0500
Subject: [infinband] cxgb4: Make c4iw_wr_log_size_order static

Message-id: <1415819300-32670-14-git-send-email-svemuri@redhat.com>
Patchwork-id: 99869
O-Subject: [RHEL7.1 PATCH BZ 1163467 13/19] RDMA/cxgb4: Make c4iw_wr_log_size_order static
Bugzilla: 1163467
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: David S. Miller <davem@redhat.com>

This fixes a sparse warning.

Signed-off-by: Steve Wise <swise@opengridcomputing.com>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit 65d4c01af0f5b394a33848bc27c4331e2959dfd5)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/hw/cxgb4/device.c b/drivers/infiniband/hw/cxgb4/device.c
index 08fb413..a2b69a4 100644
--- a/drivers/infiniband/hw/cxgb4/device.c
+++ b/drivers/infiniband/hw/cxgb4/device.c
@@ -60,7 +60,7 @@ int c4iw_wr_log = 0;
 module_param(c4iw_wr_log, int, 0444);
 MODULE_PARM_DESC(c4iw_wr_log, "Enables logging of work request timing data.");
 
-int c4iw_wr_log_size_order = 12;
+static int c4iw_wr_log_size_order = 12;
 module_param(c4iw_wr_log_size_order, int, 0444);
 MODULE_PARM_DESC(c4iw_wr_log_size_order,
    "Number of entries (log2) in the work request timing log.");
-- 
1.7.1